packages/atomic/src/utils/color-utils.ts

```diff
@@ -0,0 +1,68 @@
+export const rgbToHsv = (r: number, g: number, b: number) => {
```

Comment:
This is imported from JSUI, which itself had imported this from some stack overflow post.

It's some generic color utility function. 
We use it for the preview bar to add a nice contrast border around each hit marker (the `saturatedColor`)